go.mod

```diff
@@ -11,10 +11,10 @@ require (
 	github.com/k0kubun/pp v3.0.1+incompatible
 	github.com/leanovate/gopter v0.2.9
 	github.com/logrusorgru/aurora v0.0.0-20200102142835-e9ef32dff381
-	github.com/onflow/atree v0.4.0
+	github.com/onflow/atree v0.4.1-0.20230228213801-3c7c4769dae0
```

Comment:
Could we maybe release a new version of atree @fxamacker?

Reply:
@bastian @SupunS

This looks like a green light, so I merged Atree PR [290](https://github.com/onflow/atree/pull/290) to add DeltasSizeWithoutTempAddresses to return uncommitted slab size.

>> Could we maybe release a new version of atree @fxamacker?

Yes, unless it's needed today, I plan to tag a new version after wrapping up these items:

- Merge Supun's [PR 291](https://github.com/onflow/atree/pull/291) (currently draft) to revert PR [287](https://github.com/onflow/atree/pull/287) adding a callback to the Encoder.
- Refactor atree error handling .  I'm currently working on this so it'll wrap up soon (tomorrow or Monday if there are no surprises).  I was focused on [adding CCF codec](https://github.com/onflow/cadence/pull/2364) to onflow/cadence until yesterday.

Reply:
@SupunS https://github.com/onflow/atree/releases/tag/v0.5.0 is ready now. Could you please update the PR?

Reply:
Updated: 4470df8a9e587ff3fb662da4674ce599106d2f1d